app/public/js/main.js

```diff
@@ -32,7 +32,8 @@ var svg_background_color_online = '#0288D1',
 
 
 
-var socket = io(document.location.hostname);
+var socketHost = document.location.hostname + ":" + document.location.port;
```

Comment:
I've fixed this on my end, using just `document.location.host` which includes the port: 75451ca.